packages/hamburger-react/README.md

```diff
@@ -7,3 +7,33 @@
 ## Kom i gang
 
 [Lær hvordan du kan ta i bruk Jøkul](https://fremtind.github.io/jokul/developer/getting-started/)
+
+## Bruk av React-pakken
+
+### Installasjon
+
+1. Installér pakken med `yarn add @fremtind/jkl-hamburger-react` eller `npm i @fremtind/jkl-hamburger-react`. Stil-pakken blir automatisk installert som en avhengighet.
+2. Importér _både_ React-apkken og stil-pakken i prosjektet ditt:
+
+```js
+import { Hamburger } from "@fremtind/jkl-hamburger-react";
+import "@fremtind/jkl-hamburger/hamburger.min.css";
+```
+
+### Bruk
+
+Bruk på lys bakgrunn
+
+```jsx
+<div>
+    <Hamburger />
+</div>
+```
+
+Ved bruk på mørk bakgrunn brukes prop-en `negative` for å invertere ikonene.
+
+```jsx
+<div style={{ backgroundColor: "black" }}>
+    <Hamburger negative />
+</div>
+```
```

Comment:
Kanskje @lfbergee skal få skrive litt mer om hvordan man bruker `insideRef` for å kontrollere lukking ved klikk utenfor menyen? Kan godt gjøres i en egen PR i så fall :)

Reply:
God idé! Følte meg ikke sikker nok på det til å slenge det med i dokumentasjonen, men behovet er der absolutt